.github/workflows/integrations_test.yml

```diff
@@ -4,6 +4,7 @@ on:
    branches:
     - master
     - prepare-release
+    - nellyk-patch-1
```

Comment:
@nellyk after testing, should we remove your patch branch?

Reply:
yes this was only meant for testing. Thanks for the reminder